From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Brian Gianforcaro <b.gianfo@gmail.com>
Date: Tue, 21 Dec 2021 23:41:47 -0800
Subject: [PATCH] Remove non existent header sys/ipc.h

Serenity doesn't currently have this header, and
it doesn't appear to be needed on our platform so
remove it for the port.
---
 init.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/init.c b/init.c
index cce79e840dda5e5d1f115c0b77ddf1f95d353bdf..5d6780d3fc6f44638d0eaa5caf1d9d08a1edbe71 100644
--- a/init.c
+++ b/init.c
@@ -7,7 +7,6 @@
 #include <ctype.h>
 #include <string.h>
 #include <errno.h>
-#include <sys/ipc.h>
 #include <sys/types.h>
 #include <dlfcn.h>
 #ifdef CONFIG_VALGRIND_DEV
